src/base/memory_monitor_sr.hpp

```diff
@@ -0,0 +1,266 @@
+/*
+ *
+ * Copyright 2016 CUBRID Corporation
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+/*
+ * memory_monitor_sr.hpp - memory monitoring module header
+ */
+
+#ifndef _MEMORY_MONITOR_HPP_
+#define _MEMORY_MONITOR_HPP_
+
+#include "perf_def.hpp"
+//#include "thread_compat.hpp"
```

Comment:
Remove the un-used macro. Please check other un-used comments and macroes all together.

Reply:
I remove it all. Thanks.